app/model/commands/ActiveAssetCommand.scala

```diff
@@ -69,15 +67,21 @@ case class ActiveAssetCommand(atomId: String, youtubeId: String)
     Logger.info(s"Marking YouTube asset $youtubeId as active")
 
     getVideoStatus(youtubeId) match {
-      case response: SuccesfulYoutubeResponse => {
+      case response: SuccesfulYoutubeResponse =>
         val videoStatus = response.status
-        videoStatus match {
-          case Some("succeeded") => markAssetAsActive()
 
+        /** Processing status:
+          * failed – Video processing has failed.
+          * processing – Video is currently being processed.
+          * succeeded – Video has been successfully processed.
+          * terminated – Processing information is no longer available.
+          **/
+        videoStatus match {
+          case Some(status) if status == "succeeded" || status == "terminated" => markAssetAsActive()
```

Comment:
As far as I can tell, yes. There are a lot of videos which have the status `terminated` but you can play them.